testing/environments/args.yml

```diff
@@ -6,5 +6,5 @@ services:
     build:
       args:
         DOWNLOAD_URL: https://snapshots.elastic.co/downloads
-        ELASTIC_VERSION: 6.6.1-SNAPSHOT
+        ELASTIC_VERSION: 6.6.2-SNAPSHOT
```

Comment:
I wonder if this should actually be 6.6.3-SNAPSHOT, but I assume that isn't published yet.